polyfills/Error/isError/config.toml

```
@@ -0,0 +1,25 @@
+aliases = [ "es2025" ]
+dependencies = [
+	"_ESAbstract.CreateMethodProperty",
+	"_ESAbstract.Type",
+	"Object.prototype.toString",
+]
```

```suggestion
	"Object.prototype.toString",
	"Symbol.toStringTag",
]
```

Is already an indirect dependency, but we normally list every dependency.